packages/2d/src/lib/utils/video/parser/segment.ts

```diff
@@ -1,8 +1,11 @@
 import {MP4FileSink} from './sink';
 import {Edit} from './utils';
 
+const MAX_DECODE_QUEUE_SIZE = 30;
+
 export class Segment {
   private done: boolean = false;
```

Comment:
Let's maybe rename this to something like `currentFramePastSegmentEndTime` or something like that because I forgot what it was for and think it's easy to confuse, now that we have another "done" boolean.